### DISCUSSION TITLE
5.0.0 Alpha introduces broken changes in the arquillian config properties

### DISCUSSION THREAD

**Discussion by hantsy** [wildfly/wildfly-arquillian#201] [Q&A] [answered]
It seems the *protocol* is not supported since 5.0.0.Alpha2, and I changed to *managementProtocol*, and I set the value to `http-remoting` or `remote+http`, it will throw an exception when running the arquillian tests.

```bash
java.lang.RuntimeException: java.net.MalformedURLException: Unsupported protocol: remote+http
Caused by: java.net.MalformedURLException: Unsupported protocol: remote+http
```
See the complete log on Github actions: https://github.com/hantsy/jakartaee9-starter-boilerplate/runs/4567754712?check_suite_focus=true

**Comment by jamezp** [comment, marked as answer]
@hantsy It's likely a result of this https://issues.redhat.com/browse/WFARQ-99. You can either downgrade to 5.0.0.Alpha1 or you can add the following dependencies:
```xml
<dependency>
    <groupId>org.jboss.remoting</groupId>
    <artifactId>jboss-remoting</artifactId>
    <version>5.0.23.Final</version>
    <scope>test</scope>
</dependency>
<dependency>
    <groupId>org.jboss.remotingjmx</groupId>
    <artifactId>remoting-jmx</artifactId>
    <version>3.0.4.Final</version>
    <scope>test</scope>
</dependency>
```
I've filed https://issues.redhat.com/browse/WFARQ-102 to fix this.

Another note on that profiles surefire configuration. You should likely remove the `<java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>` system property. That should not be required unless you want to use it. In which case it should be a provided dependency

**Reply by hantsy** [reply]
@jamezp Thanks.
